docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -188,33 +196,222 @@ if (sdlManager.getSystemCapabilityManager().isCapabilitySupported(SystemCapabili
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+NSString *selectedModuleID = <#SelectedModule#>.moduleInfo.moduleID;
+```
+
+##### Swift
+```swift
+let selectedModuleID = <#SelectedModule#>.moduleInfo?.moduleId
+```
+!@
+
+### Getting Consent to Control a Module
+This feature is only available for Core v6.0+. Some OEMs may wish to ask the driver for consent before a user can control a module. The `SDLGetInteriorVehicleDataConsent` RPC will alert the driver for consent in some OEM head units if the module if not free (another user has control) and `allowMultipleAccess` (multiple users can access/set the data at the same time) is `true`. `allowMultipleAccess` is part of the `moduleInfo` in the module object.
+
+Check the `allowed` property in the `SDLGetInteriorVehicleDataConsentResponse` to see what modules can be controlled. Note the order of the `allowed` array is 1-1 with the `moduleIDs` array you passed into the `SDLGetInteriorVehicleDataConsent` RPC.
+
+!!! Note
+You should always try to get consent before setting any module data. If consent is not granted you should not attempt to set any module's data.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+SDLGetInteriorVehicleDataConsent *getInteriorVehicleDataConsent = [[SDLGetInteriorVehicleDataConsent alloc] initWithModuleType:@"<#ModuleType#>" moduleIds:@[@"<#ModuleID#>",@"<#ModuleID#>",@"<#ModuleID#>"]];
+[self.sdlManager sendRequest:getInteriorVehicleDataConsent withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if (!response.success) { return; }
+    <#Allowed is an arry of true or false values#>
+}];
+```
+##### Swift
+```swift
+let getInteriorVehicleDataConsent =  SDLGetInteriorVehicleDataConsent(moduleType: "<#ModuleType#>", moduleIds: ["<#ModuleID#>","<#ModuleID#>","<#ModuleID#>"])
+sdlManager.send(request: getInteriorVehicleDataConsent , responseHandler: { (request, response, error) in
+    guard let res = response as? SDLGetInteriorVehicleDataConsentResponse else { return }
+    let allowed = res.allowed as! [NSNumber]
+    <#Allowed is an arry of true or false values#>
+})
+```
+!@
+
+### Subscribing to Module Data 
+Seat location does not affect the ability to get data from a module. Once you know you have permission to use the remote control feature and you have the `moduleID`(s) (when connected to 6.0+ systems), you can retrieve the data for any module. The following code is an example of how to get data and subscirbe to a radio module. 
+
+When connected to < 6.0 head units, there can only be one module for each module type (e.g. there can only be one climate module, light module, radio module, etc.), so you will not need to pass a module ID.
+
+!!! NOTE
+Subscribing to the RPC must be done before sending the @![iOS]`SDLGetInteriorVehicleData`!@ @![android, javaSE, javaEE]`GetInteriorVehicleData`!@ request.
+!!!
 
 @![iOS]
```

Comment:
I don't know what you're trying to do here or why you have 3 code examples for each language. You need to explain what the code examples are and what they're doing!

Reply:
The only thing I did was move out the `SDLGetInteriorVehicleData` RPC so I can break Pre/Post 6.0 into sections. I will explain it better so devs are not confused.